app/components/appMainModule/localePageModule/electionTypePage/electionTypePageRoutes.js

```diff
@@ -5,7 +5,13 @@ function electionTypePageRoutes($stateProvider) {
   var electionTypePage = {
     name: 'appMain.localePage.electionTypePage',
     url: '/:electionYear/:electionType/:electionTypeId/:electionTitle',
-    // url: '/:electionYear/:electionType',
+    abstract: true,
+    template: '<div ui-view></div>'
```

Comment:
I don't think abstract states can have templates? Or does it just become a wrapper for the child states?